edx_solutions_api_integration/users/urls.py

```diff
@@ -38,7 +38,7 @@
         users_views.UsersPreferences.as_view(), name='users-preferences-list'),
     url(r'^(?P<user_id>[a-zA-Z0-9]+)/preferences/(?P<preference_id>[a-zA-Z0-9_]+)$',
         users_views.UsersPreferencesDetail.as_view(), name='users-preferences-detail'),
-    url(r'^(?P<user_id>[a-zA-Z0-9]+)/organizations/$',
+    url(r'^organizations/$',
```

Comment:
@msaqib52 Unless we are changing apros simultaneously, we cannot just remove the older API, because it will break Apros.

We need to:
1. Add the new `/users/organizations/?username=` endpoint first, keeping the `/:user_id/organizations/` endpoint working as well
2. Change apros to use OAuth for that API instead of API key
3. Deploy the change and make sure it's working on prod
4. Remove the old `/:user_id/organizations/` endpoint

Reply:
@bradenmacdonald we are changing apros as well to use this updated endpoint.